meta-ostro/conf/distro/include/ostroproject-ci.inc

```diff
@@ -31,6 +31,9 @@ SDKMACHINE ?= "x86_64"
 # Generate source mirror tarballs:
 BB_GENERATE_MIRROR_TARBALLS = "1"
 
+# Do not allow building of anything that hasn't been checked beforehand.
+SUPPORTED_RECIPES_CHECK = "fatal"
+
 # Configure images in "development mode".
```

Comment:
That's why I said about comment in local.conf. The less differences would be between CI builds and end-user builds - the less chances that SSTATE is not valid and less chances that we would see different errors than our users.
Another thing, does that feature support multiple lists files ? if so, then probably good idea to teach devtool to maintain own list for it's workspace as append to system-wide list.